tool/state-processor/src/staking/mod.rs

```diff
@@ -23,6 +23,7 @@ impl<S> Processor<S> {
 		{
 			let staking_ik = item_key(b"AccountMigration", b"Ledgers");
 			let deposit_ik = item_key(b"AccountMigration", b"Deposits");
+			let mut consumers = 1;
```

Comment:
This line should be put into `ledgers.into_iter().for_each(|(_, mut v)| { ...}`?

Otherwise, the `consumers`  will be at least the ledgers.len().